src/PrestaShopBundle/Resources/views/Admin/Component/Layout/employee_dropdown.html.twig

```diff
@@ -0,0 +1,73 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% if employee is defined %}
```

Comment:
employee is always defined

Reply:
And yet in the smarty template, its presence is still checked... So I guess there are use cases where it's not set yet 🤷‍♂️ We may remove this check to see if it's useful or not, although we don't have the tools to launch tests UI yet, anyway I don't mind removing this but it shouldn't be blocking anyway

Reply:
The first step was to switch from smarty to twig. That's why I'm firstly trying to get iso with the smarty template. It's tedious to convert it and then I don't check if everything it's legit to stay. Moreover we had a plan which include a refacto after.
I let you decide @jolelievre if you want to change this or not.

Reply:
Done